app/dashboard/helpers.py

```diff
@@ -467,6 +467,29 @@ def process_bounty_details(bounty_details):
     return (did_change, latest_old_bounty, latest_old_bounty)
 
 
+def record_user_action(event_name, old_bounty, new_bounty):
+    user_profile = None
+    try:
+        if event_name in ['new_bounty', 'killed_bounty', 'work_done']:
+            user_profile = Profile.objects.get(handle=new_bounty.bounty_owner_github_username)
+        if event_name in ['work_submitted']:
+            fulfillment = bounty.fulfillments.order_by('pk').first()
```

Comment:
`bounty` is undefined.

Reply:
75955a499bc59cb67ea7a914736d9678314ae391